public/tendersmash.js

```diff
@@ -430,6 +430,11 @@ app.controller("mainController", function ($scope, $http, $sce, $q, $timeout, pr
     return promises;
   }
 
+  $scope.getOrderedLists = function() {
+    var sorted = _.sortBy($scope.lists, function(x) { return x.name; });
```

Comment:
You might want to do a `.toLowerCase()` on the name here so that the sort is case-insensitive